config/v1/types.go

```diff
@@ -398,3 +398,13 @@ const (
 	// NoOpinionIncludeSubDomains means HSTS "includeSubDomains" doesn't matter to the RequiredHSTSPolicy
 	NoOpinionIncludeSubDomains IncludeSubDomainsPolicy = "NoOpinion"
 )
+
+// GroupVersionResource includes the group, version and type of resource
+type GroupVersionResource struct {
+	// Group is the API Group of the Resource.
+	Group string `json:"group" protobuf:"bytes,3,opt,name=group"`
+	// Version is the API Version of the Resource.
+	Version string `json:"version" protobuf:"bytes,4,opt,name=version"`
+	// Resource is the type that is being referenced
+	Resource string `json:"resource" protobuf:"bytes,2,opt,name=resource"`
+}
```

Comment:
We don't define the protobuf in this file yet, so I believe its not necessary here. Or?

```suggestion
// GroupVersionResource includes the group, version and type of resource
type GroupVersionResource struct {
	// Group is the API Group of the Resource.
	Group string `json:"group"`
	// Version is the API Version of the Resource.
	Version string `json:"version"`
	// Resource is the type that is being referenced
	Resource string `json:"resource"`
}
```